install.sh

```diff
@@ -991,6 +1082,7 @@ configure_collectd() {
 parse_args_wrapper "$@"
 set_variables
 determine_os
+perform_checks
```

Comment:
if you change perform_checks to perform_timedrift_check then it can be called early as it might end the install.  similar to how this script validates the api token before the determine_os call.